makefile

```diff
@@ -0,0 +1,11 @@
+lint:
+	terraform fmt --recursive
```

Comment:
watch out for `--recursive` - if you have submodules cloned then it will reach in and update all those as well and you'll have to reset or ignore them when committing.